test/unit/vl-rich-data.test.html

```diff
@@ -105,6 +114,8 @@
         });
       });
 
+      // TODO: dit test niets, de manipulatie op de pager heeft geen gevolg op de view.
+      // Hij is van in het begin al leeg en daarom slaagt de test.
```

Comment:
Dit test wel degelijk iets, er was vroeger namelijk een bug dat er undefined getoond werd wanneer de total items van de vl-pager leeg of undefined was. Het is niet omdat initieel de waarde ook 0 is, dat de test niets test.